install_config/install/advanced_install.adoc

```diff
@@ -44,9 +44,9 @@ xref:../../install_config/install/stand_alone_registry.adoc#install-config-insta
 [[advanced-before-you-begin]]
 == Before You Begin
 
-Before installing {product-title}, you must first see the xref:../../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Prerequisites] topic to
+Before installing {product-title}, you must first see the xref:../../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Prerequisites] and xref:../../install_config/install/host_preparation.adoc[Host Preparation] topics to
 prepare your hosts, which includes verifying system and environment requirements
```

Comment:
Thanks! Can you please update the path to be the following? 
xref:../../install_config/install/host_preparation.adoc#install-config-install-host-preparation[Host Preparation]

That will keep the docs builds happy :grinning: